src/components/AssetDetailComponent.jsx

```diff
@@ -104,30 +115,38 @@ export class AssetDetailComponent extends Component {
 AssetDetailComponent.propTypes = {
   loadDropDownUsers: PropTypes.func,
   allocateAsset: PropTypes.func,
+  unassignAsset: PropTypes.func,
   assetDetail: PropTypes.object,
   getAssetDetail: PropTypes.func,
   errorMessage: PropTypes.string,
   hasError: PropTypes.bool,
   isLoading: PropTypes.bool,
+  buttonLoading: PropTypes.bool,
   location: PropTypes.object,
   users: PropTypes.array,
-  newAllocation: PropTypes.array
+  newAllocation: PropTypes.object,
+  unAssignedAsset: PropTypes.object
 };
 
-const mapStateToProps = ({ asset, usersList, allocationsList }) => {
-  const { assetDetail, errorMessage, hasError, isLoading } = asset;
-  const { newAllocation } = allocationsList;
+const mapStateToProps = ({ asset, usersList }) => {
+  const {
+    assetDetail, errorMessage, hasError,
```

Comment:
NitPick: Please move each variable to it's own line. I.e.
```
const {
  assetDetail,
  errorMessage,
  hasError,
  isLoading,
  newAllocation,
  unAssignedAsset,
  buttonLoading
} = asset;

I didn't save it before

Reply:
Done!